js/bytex.js

```diff
@@ -343,21 +357,22 @@ module.exports = class bytex extends Exchange {
             const numericId = this.safeInteger (currency, 'id');
             const code = this.safeCurrencyCode (id);
             const name = this.safeString (currency, 'fullname');
-            const active = this.safeValue (currency, 'active');
+            const depositEnabled = this.safeValue (currency, 'allow_deposit');
+            const withdrawEnabled = this.safeValue (currency, 'allow_withdrawal');
+            const isActive = this.safeValue (currency, 'active');
+            const active = isActive && depositEnabled && withdrawEnabled;
             const fee = this.safeNumber (currency, 'withdrawal_fee');
             const precision = this.safeNumber (currency, 'increment_unit');
             const withdrawalLimits = this.safeValue (currency, 'withdrawal_limits', []);
             result[code] = {
-                'info': currency,
                 'id': id,
                 'numericId': numericId,
                 'code': code,
-                'type': undefined,
+                'info': currency,
```

Comment:
Can you move this back to the top position, we like to not have it in the middle